isis/tests/MatrixTests.cpp

```diff
@@ -0,0 +1,381 @@
+#include "gtest/gtest.h"
+#include "tnt/tnt_array2d.h"
+
+#include "Matrix.h"
+#include "IException.h"
+
+#include <QDebug>
+
+
+
+namespace Isis {
+
+  TEST (MatrixTests, ConstructorWithDimensions) {
+    Matrix A(2, 2, 2);
+    
+    ASSERT_EQ(2, A[0][0]);
```

Comment:
These should be EXPECT_EQ so that all failures get checked. You should also assert that the matrix has the proper dimensions.
```suggestion
    ASSERT_EQ(2, A.Rows());
    ASSERT_EQ(2, A.Columns());
    EXPECT_EQ(2, A[0][0]);
```